include/jwt-cpp/base.h

```diff
@@ -116,30 +111,31 @@ namespace jwt {
 				}
 			};
 
-			inline padding count_padding(const std::string& base, const std::vector<std::string>& fills) {
-				for (const auto& fill : fills) {
-					if (base.size() < fill.size()) continue;
+			template<class StrInputIt>
+			padding count_padding(string_view base, StrInputIt fillStart, StrInputIt fillEnd) {
```

Comment:
Could you please change these to snake_case to match the rest of the code base?

Reply:
Yes sure.